src/lib/components/forms/form/form-components/number-input/number-input.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Input } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { NumberInputInterface } from './number-input.interface';
+import { noop } from 'rxjs';
+
+@Component({
+    selector: 'number-input',
+    templateUrl: './number-input.component.html',
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: NumberInputComponent,
+            multi: true
+        }
+    ]
+})
+export class NumberInputComponent implements ControlValueAccessor, NumberInputInterface {
+
+    /** @description If true, the button will be disabled. Default false. */
+    @Input()
+    public isDisabled: boolean;
```

Comment:
Initialisieren? Ich erinnere dich quasi an deine Worte aus dem Planning 😛 
```suggestion
    public isDisabled: boolean = false;
```